etcd/etcd-live-cluster-reconfiguration.md

```diff
@@ -1,236 +1,350 @@
 
-```
-added member 9bf1b35fc7761a23 to cluster
+1. Run the `etcdctl member add` command.
 
-ETCD_NAME="node4"
-ETCD_INITIAL_CLUSTER="1dc800dbf6a732d8839bc71d0538bb99=http://10.0.1.1:2380,f961e5cb1b0cb8810ea6a6b7a7c8b5cf=http://10.0.1.2:2380,8982fae69ad09c623601b68c83818921=http://10.0.1.3:2380,node4=http://10.0.1.4:2380"
-ETCD_INITIAL_CLUSTER_STATE=existing
-```
+   For example:
 
-The `ETCD_DISCOVERY` environment variable defined in `20-cloudinit.conf` conflicts with the `ETCD_INITIAL_CLUSTER` setting needed for these steps, so the first step is clearing it by overriding `20-cloudinit.conf` with a new drop-in, `99-restore.conf`. `99-restore.conf` contains an empty `Environment="ETCD_DISCOVERY="` string.
+    ```sh
+    $ etcdctl member add node4 http://0.0.0.4:2380
+    ```
 
-The complete example looks like this. On the `node4` Container Linux host, create a  temporary systemd drop-in, `/run/systemd/system/etcd2.service.d/99-restore.conf` with the contents below, filling in the information from the output of the `etcd member add` command we ran previously:
+    The output of a successful member addition is given below:
 
-```ini
-[Service]
-# remove previously created proxy directory
-ExecStartPre=/usr/bin/rm -rf /var/lib/etcd2/proxy
-# NOTE: use this option if you would like to re-add broken etcd member into cluster
-# Don't forget to make a backup before
-#ExecStartPre=/usr/bin/rm -rf /var/lib/etcd2/member /var/lib/etcd2/proxy
-# here we clean previously defined ETCD_DISCOVERY environment variable, we don't need it as we've already bootstrapped etcd cluster and ETCD_DISCOVERY conflicts with ETCD_INITIAL_CLUSTER environment variable
-Environment="ETCD_DISCOVERY="
-Environment="ETCD_NAME=node4"
-# We use ETCD_INITIAL_CLUSTER variable value from previous step ("etcdctl member add" output)
-Environment="ETCD_INITIAL_CLUSTER=node1=http://10.0.1.1:2380,node2=http://10.0.1.2:2380,node3=http://10.0.1.3:2380,node4=http://10.0.1.4:2380"
-Environment="ETCD_INITIAL_CLUSTER_STATE=existing"
-```
+    ```sh
+    added member 9bf1b35fc7761a23 to cluster
 
-Run `sudo systemctl daemon-reload` to parse the new and edited units. Check whether the new [drop-in][drop-in] is valid by checking the service's journal: `sudo journalctl _PID=1 -e -u etcd2`. If everything is ok, run `sudo systemctl restart etcd2` to activate your changes. You will see that the former proxy node has become a cluster member:
+    ETCD_NAME="node4"
+    ETCD_INITIAL_CLUSTER="1dc800dbf6a732d8839bc71d0538bb99=http://10.0.1.1:2380,f961e5cb1b0cb8810ea6a6b7a7c8b5cf=http://10.0.1.2:2380,8982fae69ad09c623601b68c83818921=http://10.0.1.3:2380,node4=http://10.0.1.4:2380"
+    ETCD_INITIAL_CLUSTER_STATE=existing
+    ```
+2. Store the output of this command for later use.
 
-```
-etcdserver: start member 9bf1b35fc7761a23 in cluster 36cce781cb4f1292
-```
+3. On the `node4` Container Linux host, create a  temporary systemd drop-in, `/run/systemd/system/etcd-member.service.d/99-restore.conf`.
```

Comment:
steps 3 and 4 should be combined since this isn't using systemd drop-ins any more:
```
3. Use the information from `etcdctl member add` to provision a new Container Linux host with the following Container Linux Config:
(yaml)
```